api/proto/core/pipeline/definition/definition.proto

```diff
@@ -138,20 +138,22 @@ message PipelineDefinitionListRequest {
   int64 pageNo = 2;
   string location = 3;
   string name = 4;
-  repeated string creator = 5;
-  repeated string executor = 6;
-  repeated string category = 7;
-  repeated string ref = 8;
-  repeated string remote = 9;
-  repeated string timeCreated = 10;
-  repeated string timeStarted = 11;
-  repeated string status = 12;
-  repeated string idList = 13;
-  repeated string ascCols = 14;
-  repeated string descCols = 15;
-  int64 totalActionNum = 16;
-  int64 ExecutedActionNum = 17;
-  repeated string sourceIDList = 18;
+  string accurateName = 5;
```

Comment:
I suggest:
`name` is accurate, and `fuzzyName` is fuzzy.

Reply:
done